src/tsolvers/LATHandler.cc

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Antti on 21.10.21.
```

Comment:
Maybe use our new cool way of licensing? :)

Reply:
Fixed :)